com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md

```diff
@@ -37,31 +27,45 @@ There are three tabs that you can use to set up your HDRP Project for different
 * [HDRP + VR](#VRTab): Use this tab to set up your HDRP Project and enable support for virtual reality.
 * [HDRP + DXR](#DXRTab): Use this tab to set up your HDRP Project and enable support for ray tracing.
 
+Also every configuration is separated in two scopes:
+
+- **Global:** settings in Unity Editor or in the [HDRP Asset](HDRP-Asset.md) set in Graphics Settings.
```

Comment:
Could be changed to say
`settings in Unity Editor, [HDRP Global Settings](Default-Settings-Window.md), or Graphics Settings'  [HDRP Asset](HDRP-Asset.md)`